python/cudf/cudf/_lib/column.pyx

```diff
@@ -155,7 +155,11 @@ cdef class Column:
             )
 
         if value is not None:
-            required_size = bitmask_allocation_size_bytes(self.base_size)
+            # base_size may be zero for a struct column with no
+            # children, but the column may have non-zero length.
```

Comment:
Wait.. I thought that the length of a column could only ever be <= its `base_size`?

Reply:
```
In [1]: import cudf
cudf.Se
In [2]: cudf.Series([None, None, None, None], dtype=cudf.StructDtype({}))
Out[2]: 
0    None
1    None
2    None
3    None
dtype: struct

In [3]: series = cudf.Series([None, None, None, None], dtype=cudf.StructDtype({}))

In [4]: series._column.base_size
Out[4]: 0

In [5]: series._column._size
Out[5]: 4

In [6]: len(series._column)
Out[6]: 4

In [7]: series._column.base_children
Out[7]: ()
```

Reply:
It's not a particularly interesting column (it's a very heavyweight way of representing a bitmask), but it's constructable.